packages/documentation/src/stories/foundation/metadata/metadata-example.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head prefix="og: https://ogp.me/ns#">
+    <!--Content Security Policy-->
+    <meta
+      http-equiv="Content-Security-Poilicy"
+      content="default-src self; connect-src https://unpkg.com/"
```

Comment:
Usually, unpkg should not be used for performance reasons.

```suggestion
      content="default-src self;"
```

Reply:
In the Icons documentation we tell our users to use connect.src htps.//unpkg.com/ because the icons are loaded from there apparently.

Reply:
They are available through unpkg because they are static files that are published as an NPM package. It's a fallback that's used in the icon component, but performance is generally poor based on some tests.

On the icons getting started page is also a warning, that loading icons from a CDN should not be used in production.

Generally allow-listing unpkg could be a potential security risk for unsuspecting applications because it's really easy to publish malicious code to npm.